src/Veldrid.ImageSharp/ImageSharpCubeMapTexture.cs

```diff
@@ -0,0 +1,105 @@
+﻿using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Advanced;
+using SixLabors.ImageSharp.PixelFormats;
+using System;
+using System.Runtime.InteropServices;
+
+namespace Veldrid.ImageSharp
+{
+    public class ImageSharpCubeMapTexture

```

Comment:
Other parts of Veldrid's documentation spell this as "Cubemap" ("map" not being capitalized). For consistency, we should do that here.